### PR TITLE
Fixed size index

### PR DESCRIPTION
**Pull request by brian-brazil**
Per RFC 1212 4.1.6(2)

@SuperQ

### CODE REVIEW

File: generator/FORMAT.md [hunk 1]
```diff
@@ -32,6 +32,8 @@ module_name:
      indexes:
       - labelname: ifIndex
         type: gauge
+        fixed_size: 0   # Only possible for OctetString/DisplayString.
```
Comment (SuperQ) [inline generator/FORMAT.md]: I would suggest making this a separate example, to make it more clear that this is for `type: DIsplayString`.  Also, comment that it's an optional field.

Comment (brian-brazil) [reply generator/FORMAT.md]: Done

**Comment by brian-brazil** [comment]
Looks like this doesn't work on OSX for some reason I can't determine, I'm presuming some compiler oddity there as netsnmp has a NULL ranges yet snmptranslate shows the range. This all works fine on Ubuntu.

**Comment by brian-brazil** [comment]
It also works on Red Hat, so I'm going with OSX weirdness.

**Pull request merged**